snakegame/requirements.txt

```
@@ -1 +1,3 @@
-reflex>=0.2.0
\ No newline at end of file
+#reflex>=0.2.9
+# temporary until state.reset() bug is fixed
+git+https://github.com/reflex-dev/reflex@masenf/state-reset-deepcopy
```

We can remove the temporary change now.
```suggestion
reflex>=0.2.9
```